### ISSUE TITLE
Upload photos

### ISSUE THREAD

**Issue by davidpdrsn** [davidpdrsn/CardistryIO#30]

**Comment by bendobos** [comment]
People came up to me at CC and asked for this specific feature. We should consider it for the beta, if it's not too much work to implement.

**Comment by davidpdrsn** [comment]
I don't think we can make it for the beta. It's quite a bit work. Have to find to host the photos figure out uploading from mobile devices, etc.

I think it would make sense to look into this before we do video upload, as a way of getting our feet wet with user uploaded content.
